Create 2024-06-05-TAC.md

maximilien opened the pull request:
Initial minutes with Agenda

maximilien committed:
Create 2024-06-05-TAC.md

Initial minutes with Agenda

Signed-off-by: dr.max <maxim@us.ibm.com>

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+
+* [ ] Norman Ashley, Cisco
+* [ ] Michael (Max)imilien, IBM
+* [ ] Sam Stanwyck, NVIDIA
```
Yarkin Doroz - attended

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+* [ ] Michael (Max)imilien, IBM
+* [ ] Sam Stanwyck, NVIDIA
+* [ ] Sophie Schmieg, Google
+* [ ] Brian Jarvis, Amazon Web Services Inc.
```
Attended

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+* [ ] Brian Jarvis, Amazon Web Services Inc.
+* [ ] Thomas Bailleux, SandboxAQ
+
+## Additional attendees
```
Alex Bozarth, Bryan Uhri, Hart Montgomery, Nigel Jones, Ry Jones

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+2. Misc
+  - Workgroups
+  - OS summit conference
+  - Blogs
```
Maximilen updated the group that the blog submitted by him and Ashwin is now ready for review. Encouraged others to be sure to submit blogs and content.

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+1. New business
+2. Misc
+  - Workgroups
+  - OS summit conference
```
Maximilien noted to the group that a PQCA session was submitted for OSS EU in Vienna. 

Invited to give a talk at ICML, but someone else will need to attend as the conferences are on the same day in different locations.

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+0. Finalize agenda
+1. New business
+2. Misc
+  - Workgroups
```
Maximilien led a working group discussion, noting the initial working groups are: Doc, Algorithms, and Security. 

### Docs Update
Jones noted that he has started creating documents for PQ Code Package but asked if should this be implemented at the TAC level. Washington noted that documents at the TAC level are generally used for guidance and accepted at the project level. 

Maximilien noted that to wait is to ensure the process of creating docs is consistent across the alliance and projects. Jones noted that for Hyperledger they use Mockdocs and suggested to use that for this. Just to get something started and work from there. 

Jones to share a Hyperleder document that can be used as a template. 

2 parts to documentation: 1. Process and tooling 2. content

**Next Steps:** Jones will take the lead on starting the process and using the Hyperleder/PQCP document process. 

### Security Update
Maximilien noted that at the last TAC meeting, Dana presented OpenSSF Security Best Practices. Noting that the scorecard process has been started in OQS but they are still mitigating issues. After that has been completed, it will be rolled out to other projects in OQS and PQ Code Package. 

Jones noted that the community's concern with the scorecard is visibility without resolving all the issues. The community agreed to fix the issues before making the scorecard public. 

Montgomery noted that the alliance needs a well-documented security vulnerability disclosure process.

### CBOM Update
Maximilien requested this working group to be created and the request came from an internal request stating that CBOMs and SBOMs need to be created from the source. Need to have a catalog of CBoMs, create new ones where they don't exist, and encourage others to do the same. Jones and Maximilen have started research for creating CBoMs for projects. 

Have looked into how to create a catalog of existing CBoMs and what it takes to create a catalog. For example Kubernetes and seeing if we have a list of CBoMs for Kubernetes. This invites the question of introducing tooling to have these updated as well. 

**Next Steps:** At the next meeting Maximilien and Jones will provide an update and findings to 1.) determine if the TAC is interested, 2.) does TAC members have CBoMs that they are maintaining, and 3.) if they have documentation on how they are managed. 

The first step is to collect as much information before we create the working group. 
Uhri noted that the NIST PQC Discovery is pushing for S/CBOM output and for them to create documentation and tooling.

**AIs:** 
- [ ] Jones will see about An open source c/sbom generator

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+  - Blogs
+  - Vice-chair election 
+  - Action Items
+3. Cadence of this meeting
```
Maximilien proposed to move this meeting to monthly. Discussion ensued about the risk of moving the TAC meeting to monthly.

It was decided to keep these meetings every 2 weeks and then cancel when we don't have enough content to host a meeting.

Naomi-Wash commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,72 @@
+
+ ## New
+
+ * [ ] Lifecycle document completion [Hart]
```
It was reminded that the lifecycle document needs to be reviewed and we will use the next meeting to review and live edit as a group.

maximilien committed:
Update 2024-06-05-TAC.md

Added changes from @Naomi-Wash and a few from me.

@ryjones @planetf1 @Naomi-Wash and @brian-jarvis-aws please peruse and give me thumbs up or final suggestions before I merge. Thanks.

Signed-off-by: dr.max <maxim@us.ibm.com>

maximilien commented:
Hi, all, see latest commit which includes @Naomi-Wash’s notes and a few from me.

Give me a quick 👍 or suggestions one of you: @planetf1 @ryjones @brian-jarvis-aws 

Thanks

brian-jarvis-aws commented on meeting-minutes/2024/2024-06-05-TAC.md:
```diff
@@ -0,0 +1,134 @@
+		- Docs Update
+				Nigel noted that he has started creating documents for PQ Code Package but asked if should this be implemented at the TAC level. Naomi noted that documents at the TAC level are generally used for guidance and accepted at the project level.
+
+				Max noted that to wait is to ensure the process of creating docs is consistent across the alliance and projects. Nigel noted that for Hyperledger they use Mockdocs and suggested to use that for this. Just to get something started and work from there.
```
I think it was MkDocs (Mk I believe being short for Markdown), not Mockdocs.

brian-jarvis-aws reviewed (commented):
Just one minor comment about the name of the documentation tool.
